pkg/forwarder/transaction/transaction.go

```diff
@@ -23,21 +23,25 @@ import (
 )
 
 var (
-	connectionDNSSuccess               = expvar.Int{}
-	connectionConnectSuccess           = expvar.Int{}
-	transactionsExpvars                = expvar.Map{}
-	transactionsInputBytesByEndpoint   = expvar.Map{}
-	transactionsConnectionEvents       = expvar.Map{}
-	transactionsInputCountByEndpoint   = expvar.Map{}
-	transactionsDropped                = expvar.Int{}
-	transactionsDroppedByEndpoint      = expvar.Map{}
-	transactionsDroppedOnInput         = expvar.Int{}
-	transactionsRequeued               = expvar.Int{}
-	transactionsRequeuedByEndpoint     = expvar.Map{}
-	transactionsRetried                = expvar.Int{}
-	transactionsRetriedByEndpoint      = expvar.Map{}
-	transactionsRetryQueueSize         = expvar.Int{}
-	transactionsSuccessByEndpoint      = expvar.Map{}
+	// ForwarderExpvars is the root for expvars in the forwarder.
+	ForwarderExpvars = expvar.NewMap("forwarder")
+
```

Comment:
Why is this in `transaction` and not `forwarder` ? It's the "forwarder" expvar after all.

Should we add a `forwarder/telemetry` package if it's beacause of cyclic import

Reply:
`ForwarderExpvars` is public and in `pkg/forwarder/transaction` because it is used by both `pkg/forwarder` and `pkg/forwarder/internal/retry`.
Splitting the expvars  was not trivial as some of them are used in difference files (Kind a lack of encapsulation).

Reply:
What about `pkg/forwarder/telemetry` or `forwarder/internal/telemetry` ? We could then access it from both `pkg/forwarder` and `pkg/forwarder/internal/retry`

Reply:
First, there is a lack of encapsulation for several expvars in the forwarder  (The ones that are used at different locations). Ideally, an expvar should be used in the single file. Creating a package for all the expvars makes thing worth.

I tried to make the minimal changes without introducing more issues than before. 

This part can definitely be improved but is clearly out of the scope of this PR.